src/rapids_pre_commit_hooks/alpha_spec.py

```
@@ -48,16 +51,37 @@ def strip_cuda_suffix(args: argparse.Namespace, name: str) -> str:
 
 
 class AlphaSpecHandler(Handler):
+    @dataclasses.dataclass
+    class PackagesContext:
+        packages_is_reference_anchor: bool
+
     def __init__(self, linter: Linter, args: argparse.Namespace):
```

Ruff (0.16.1)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)